docker/1.3.0/final/Dockerfile.gpu

```diff
@@ -50,6 +50,8 @@ RUN framework_installable_local=$(basename $framework_installable) && \
 RUN pip install onnx==1.2.1 \
                 keras-mxnet==2.2.2
 
+COPY keras.json /root/.keras/keras.json
```

Comment:
This file is pretty short. Can we just create it here? So we don't need to worry about where to put it.

Reply:
good idea